_posts/en/newsletters/2020-08-26-newsletter.md

```diff
@@ -0,0 +1,110 @@
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.11.0-beta][lnd 0.11.0-beta] is now released.   This new major
+  version allows accepting [large channels][topic large channels] (by
+  default, this is off) and contains numerous improvements to its
+  backend features that may be of interest to advanced users (see the
+  [release notes][lnd 0.11.0-beta]).
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #14582][] and [#19743][Bitcoin Core #19743] wallet: always do partial spends FIXME:jonatack
+
+- [Bitcoin Core #19550][] adds a new `getindexinfo` RPC that lists each
+  optional index that has been enabled, whether it is fully synced to
+  the tip of the block chain, and (in case it hasn't been fully synced)
+  how many blocks have been indexed so far.
+
+- [C-Lightning #3954][] updates both the `fundpsbt` and `utxopsbt` RPCs
+  so that they can each take a `locktime` parameter that specifies the
+  nLockTime of the transaction to create.  The PR notes that this is
+  "required for [dual funding][bolts #524], where the opener sets [the
+  locktime]".
+
+- [BIPs #955][] BIP174: add hash preimage fields to inputs FIXME:dongcarl
+
+- [BOLTs #688][] adds support for [anchor outputs][topic anchor outputs]
+  to the LN specification.  This extends commitment transactions with
+  two extra outputs---one for each counterparty---which can be used for
+  Child Pays For Parent ([CPFP][topic cpfp]) fee bumping.   With this
+  change, it becomes possible to fee bump commitment transactions that
+  may have been signed days or weeks earlier, when the current feerates
+  would've been hard to predict.  In practice, LN nodes using anchor
+  outputs should pay lower fees normally (because there's no longer any
+  incentive to overestimate fees) and also have greater security
+  (because, if feerates do increase beyond what was predicted, the node
+  can fee bump its commitment transaction).  Various degrees of support
```

Comment:
Suggest breaking into two sentences to match the two benefits of anchor outputs:
```suggestion
  would've been hard to predict.  In practice, LN nodes using anchor
  outputs should normally pay lower fees because there's no longer any
  incentive to overestimate fees. Anchor outputs also provide greater security
  because, if feerates do increase beyond what was predicted, the node
  can fee bump its commitment transaction.  Various degrees of support
```